src/JustSaying/MessagingConfig.cs

```diff
@@ -21,6 +21,7 @@ public MessagingConfig()
     public int PublishFailureReAttempts { get; set; }
     public TimeSpan PublishFailureBackoff { get; set; }
     public Action<MessageResponse, Message> MessageResponseLogger { get; set; }
+    public Action<MessageBatchResponse, IEnumerable<Message>> MessageBatchResponseLogger { get; set; }
```

Comment:
I think this would probably be better with `ICollection<T>` or `IList<T>` as well so consumers can access the count easily (as we already know it internally).